nextroute/common/fast_haversine.go

```diff
@@ -38,7 +38,7 @@ func wrap(deg float64) float64 {
 
 // Distance returns the distance between two locations in meters.
 func (f FastHaversine) Distance(from, to Location) float64 {
```

Comment:
Why does Haversine throw an error and FastHaversine return zero/

Reply:
🤷🏻 Haven't been thorough with the code yet 😅 . I'll return 0 and an error.